packages/server/src/fhir/operations/dbcolumnstatisticslookup.ts

```diff
@@ -0,0 +1,276 @@
+import { allOk, badRequest, OperationOutcomeError } from '@medplum/core';
+import { FhirRequest, FhirResponse } from '@medplum/fhir-router';
+import { OperationDefinition } from '@medplum/fhirtypes';
+import { Client, Pool } from 'pg';
+import { requireSuperAdmin } from '../../admin/super';
+import { DatabaseMode, getDatabasePool } from '../../database';
+import { escapeUnicode } from '../../migrations/migrate-utils';
+import { isValidTableName } from '../sql';
+import { buildOutputParameters, parseInputParameters } from './utils/parameters';
+
+const LookupOperation: OperationDefinition = {
+  resourceType: 'OperationDefinition',
+  name: 'db-column-statistics-lookup',
+  status: 'active',
+  kind: 'operation',
+  code: 'lookup',
+  experimental: true,
+  system: true,
+  type: false,
+  instance: false,
+  parameter: [
+    {
+      use: 'in',
+      name: 'tableName',
+      type: 'string',
+      min: 0,
+      max: '1',
+    },
+    {
+      use: 'out',
+      name: 'defaultStatisticsTarget',
+      type: 'integer',
+      min: 1,
+      max: '1',
+    },
+    {
+      use: 'out',
+      name: 'table',
+      part: [
+        { use: 'out', name: 'tableName', type: 'string', min: 1, max: '1' },
```

Comment:
You shouldn't need to specify `Parameters.parameter.part.use`, since it's implicitly the same as its parent parameter's `use`